src/components/DataTable/DataTable.tsx

```diff
@@ -98,8 +87,12 @@ export function DataTable<TData, TValue>({
                 <TableCell>
                   {table.getColumn('status') && row.getValue('status') === ContactStatus.Awaiting ? (
                     <div className="grid grid-cols-2 items-center w-fit gap-4 ">
-                      {AcceptDialog ? <AcceptDialog row={row} /> : null}
-                      {RejectDialog ? <RejectDialog row={row} /> : null}
+                      {isContact(row.original) && row.original.status === ContactStatus.Awaiting && (
+                        <>
+                          <ContactAcceptDialog row={row as Row<Contact>} />
+                          <ContactRejectDialog row={row as Row<Contact>} />
```

Comment:
This should be replaced with something similar to renderItem, for example `renderInlineActions`

Reply:
Added a prop